base/abstractset.jl

```diff
@@ -25,7 +25,7 @@ This is an infix operator, allowing `s ∪ itr`.
 See also [`unique`](@ref), [`intersect`](@ref), [`isdisjoint`](@ref), [`vcat`](@ref), [`Iterators.flatten`](@ref).
 
 # Examples
-```jldoctest
+```jldoctest; filter = r"(?m)^\\s+\\S"
```

Comment:
Maybe good to test for the order here. As I understand the docstring, the order is preserved - though it's not clear to me if the order between the elements of the two argument are preserved (i.e. is `1 3 2` a valid order?)

Reply:
only problem is that I think `union(Set([1, 2]), 2:3)` will not be ordered. I agree that the others in this doctest are though. not sure the best way to filter that